deploy.sh

```diff
@@ -40,20 +46,16 @@ echo -e "Generated docs\n"
 # Change directory to $DOCS_DIR
 cd $DOCS_DIR
 
-# Flatten content of $DOCS_DIR
-mv $PACKAGE_NAME/**/** ./
```

Comment:
![image](https://cloud.githubusercontent.com/assets/441011/20156553/0ee3d6f4-a6d1-11e6-989e-9742d341d70e.png)

That was how it looked.
Ist now flat again by default.